easybuild/easyconfigs/b/BioKanga/BioKanga-3.4.5-foss-2015b.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'BioKanga'
+version = '3.4.5'
+
+homepage = 'http://sourceforge.net/projects/biokanga'
+description = """BioKanga is an integrated toolkit of high performance bioinformatics
+  subprocesses targeting the challenges of next generation sequencing analytics.
+  Kanga is an acronym standing for 'K-mer Adaptive Next Generation Aligner'."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+
+sources = ['buildbiokanga_%s.zip' % version.replace('.', '_')]
+
```

Comment:
please drop this empty lines, we usually keep `sources` and `source_urls` close to each other